MetaMask/metamask-mobile
# feat: Update the Palm logo with a new one

## **Description**

The palm logo needed updating, it was confirmed that the logo used in the extension was the correct one.

## **Related issues**

Fixes:

## **Manual testing steps**

In the header, select the network to bring up the change network bottom list, click on Add network, the Palm logo should be visible next to the Palm network (if the network is part of the networks that can be added).

## **Screenshots/Recordings**

| With old logo  | The new logo | With new logo  |
|----------|----------|----------|
| ![Image 1](https://github.com/MetaMask/metamask-mobile/assets/165834542/fa0377a9-a450-48ab-b122-a43a9c269c32) | ![Image 2](https://github.com/MetaMask/metamask-mobile/assets/165834542/5250f59a-3ea0-41c0-afa1-c115b41e8a53) | ![Image 3](https://github.com/MetaMask/metamask-mobile/assets/165834542/98150829-95a8-4e40-9419-708087015fa9) |




### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
